packages/round-manager/src/features/round/ViewApplicationPage.tsx

```diff
@@ -464,70 +732,110 @@ export default function ViewApplicationPage() {
+                    className="flex flex-row items-center"
+                    data-testid="project-createdAt"
+                  >
+                    <CalendarIcon className="h-4 w-4 mr-2 text-grey-400" />
+                    <span className="text-sm text-grey-400">
+                      Created on:{" "}
+                      {application?.project?.createdAt
+                        ? formatDateWithOrdinal(
+                            new Date(Number(application?.project?.createdAt))
+                          )
+                        : "-"}
                     </span>
-                  </div>
-
-                  <hr className="my-6" />
-
-                  <h2 className="text-xs mb-2">Description</h2>
-
-                  <p
-                    dangerouslySetInnerHTML={{
-                      __html: renderToHTML(
-                        application?.project?.description.replace(
-                          /\n/g,
-                          "\n\n"
-                        ) ?? ""
-                      ),
-                    }}
-                    className="text-md prose prose-h1:text-lg prose-h2:text-base prose-h3:text-base prose-a:text-blue-600"
-                  ></p>
-
-                  <hr className="my-6" />
-
-                  {answerBlocks &&
-                    answerBlocks?.map((block: AnswerBlock) => {
-                      const answerText = Array.isArray(block.answer)
-                        ? block.answer.join(", ")
-                        : block.answer ?? "";
-
-                      return (
-                        <div key={block.questionId} className="pb-5">
-                          <h2 className="text-xs mb-2">{block.question}</h2>
-                          {block.type === "paragraph" ? (
-                            <p
-                              dangerouslySetInnerHTML={{
-                                __html: renderToHTML(
-                                  answerText.replace(/\n/g, "\n\n")
-                                ),
-                              }}
-                              className="text-md prose prose-h1:text-lg prose-h2:text-base prose-h3:text-base prose-a:text-blue-600"
-                            ></p>
-                          ) : (
-                            <p className="text-base text-black">
-                              {answerText.replace(/\n/g, "<br/>")}
-                            </p>
-                          )}
-                        </div>
-                      );
-                    })}
+                  </span>
                 </div>
-                <div className="sm:basis-1/4 text-center sm:ml-3"></div>
+                <hr className="my-6" />
+                {/* Description */}
+                <h2 className="text-xs mb-2">Description</h2>
+                <p
+                  dangerouslySetInnerHTML={{
+                    __html: renderToHTML(
+                      application?.project?.description.replace(
```

Comment:
Description should be sanitized or we're creating an opportunity for injection attacks here.